scrnaseq.Rmd

```diff
@@ -1462,7 +1464,9 @@ knitr::kable(enriched_top, align="l", caption="Top enriched term per geneset") %
   kableExtra::scroll_box(width="100%", height="700px")
 ```
 
-# Loupe Cell Browser integration
+# Further analysis with other tools
+
+## Loupe Cell Browser integration
```

Comment:
Can we make the headers for Loupe and Cerebro equivalent? 
Export to ... or ... integration.

Reply:
Done